proto/services/updates_svc/v1/updates_svc.proto

```diff
@@ -6,7 +6,9 @@ option go_package = "gen/services/updates-svc/v1";
 
 // An event that gets raised on non-event sourced entities
 message EntityEvent {
-  string entity_id = 1;
+  string event_type = 1;
```

Comment:
again, why is this split needed? why would a client even care about this impl. detail?

Reply:
Any good wording to not use "entity" or "aggregate"?

Reply:
I don't think I made myself clear.

Why do we, additionally to "normal" Events also have EntityEvents?

Reply:
To differentiate between non event-sourced and event-sourced entities. But yes, I dont want to show this implementation detail to the client either. But I dont have a good abstraction for the response ... lets strich with only entities ...

Reply:
why is any abstraction needed at all? just return a normal event

Reply:
Its not about the event. Its about the entity or aggregate.

Reply:
yes, just pretend its an aggregate